android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/listener/Callbacks.kt

```diff
@@ -85,6 +87,14 @@ class Callbacks {
         }
     }
 
+    fun setOnError(onErrorListener: OnErrorListener?) {
+        this.onErrorListener = onErrorListener
+    }
+
+    fun getOnError(): OnErrorListener? {
+        return onErrorListener
+    }
+
```

Comment:
This is java like programming with getter/setters you can just make the property `onErrorListener` public

Reply:
Done